src/Features/Core/Portable/AddConstructorParametersFromMembers/State.cs

```diff
@@ -1,34 +1,27 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.Linq;
-using System.Threading;
 using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.Text;
+using Microsoft.CodeAnalysis.PooledObjects;
 
 namespace Microsoft.CodeAnalysis.AddConstructorParametersFromMembers
 {
     internal partial class AddConstructorParametersFromMembersCodeRefactoringProvider
     {
         private class State
         {
-            public TextSpan TextSpan { get; private set; }
-            public IMethodSymbol MatchingConstructor { get; private set; }
             public IMethodSymbol DelegatedConstructor { get; private set; }
```

Comment:
note: 'DelegatedConstructor' isn't a great name, now that you've removed MatchingConsturctor.  Indeed, it feels like you should use the term MatchingConstructor  and remove DelegatedConstructor , or just name this 'ConstructorToAddTo'.